benchmark/buffers/buffer-copy.js

```diff
@@ -0,0 +1,18 @@
+'use strict';
+const common = require('../common.js');
+
+const bench = common.createBenchmark(main, {
+  size: [10, 1024, 2048, 4096, 8192],
```

Comment:
Please also add some significantly bigger entries.

Reply:
We'll want to go up into the MB range, maybe like 16 or 32mb.